### DISCUSSION TITLE
Closed, ignore!

### DISCUSSION THREAD

**Discussion by DevBadger10** [olive-editor/olive#2128] [Q&A] [answered]
I am a previous 0.1 user switching to 0.2. How do I get a 0.1 project into a 0.2 project? And side-question, is there a way to use the legacy effects editor?
![image](https://user-images.githubusercontent.com/104037142/209078178-689b93da-6141-4c55-bfd4-c5f267930b06.png)

Nvm i just exported a video and used a different project to continue editing

**Comment by DevBadger10** [comment, marked as answer]
Nvm i just exported a video and used a different project to continue editing

**Comment by TheTechRobo** [comment]
> And side-question, is there a way to use the legacy effects editor?

That's supposed to be done by the nodes, but it's kinda complicated to do large effects

**Reply by DevBadger10** [reply]
Yeah. I can see how the nodes are meant to be used, but I personally prefer the simplicity  of just clicking a few buttons then boom. Its a lot less customizable, but for the time being, I'm gonna use the legacy editor. But I will try to learn how to recreate effects with nodes in 0.2.
